tests/raftstore/cluster.rs

```diff
@@ -458,6 +461,63 @@ impl<T: Simulator> Cluster<T> {
         assert!(status_resp.has_region_detail());
         status_resp.take_region_detail()
     }
+
+    pub fn partition(&mut self, s1: &HashSet<u64>, s2: &HashSet<u64>) {
+        for node_id in s1 {
+            let filter = new_partition_filter(s2);
+            self.sim.wl().push_trans_filter(*node_id, filter);
+        }
+        for node_id in s2 {
+            let filter = new_partition_filter(s1);
+            self.sim.wl().push_trans_filter(*node_id, filter);
+        }
+    }
+
+    pub fn heal_partition(&mut self, s1: &HashSet<u64>, s2: &HashSet<u64>) {
```

Comment:
so that the API is more flexible...
if we have A partition from B, and C partition for D meanwhile, and we just want to heal the partition A/B

but it may cause trouble if mis-used